samples/net/ws_echo_server/README.rst

```diff
@@ -0,0 +1,36 @@
+.. _websocket-server-sample:
+
+Websocket Server
+################
+
+Overview
+********
+
+The websocket-server sample application for Zephyr implements a websocket
+server. The websocket-server listens for incoming IPv4 or IPv6 HTTP(S)
+requests.
```

Comment:
Add what it does in response to the incoming requests.
It also appears you need to use a web browser to interact with this sample, but you don't talk about that here.